frame/contracts/src/rent.rs

```diff
@@ -330,11 +332,16 @@ where
 
 		// Enact the verdict only if the contract gets removed.
 		match verdict {
-			Verdict::Evict { .. } => {
+			Verdict::Evict { ref amount } => {
+				let rent_payed = amount
```

Comment:
minor note: I would rather name it total_rent because the `amount` is actually not payed, isn't it ?

Reply:
Added a comment that should make clear that this indeed the `rent_payed`.

Reply:
ah I missunderstood `OutstandingAmount`. yes makes sense now